pydicer/convert/pet_to_nifti_convert_new.py

```diff
@@ -0,0 +1,409 @@
+import warnings
+from os.path import join
+from datetime import time, datetime
+
+import numpy as np
+from skimage.draw import polygon
+import SimpleITK as sitk
+import pydicom as pdcm
+from pydicom.tag import Tag
+
+
+def convert_dicom_to_nifty(input_filepaths,
+                           output_folder,
+                           modality='CT',
+                           sitk_writer=None,
+                           rtstruct_file=None,

```

Comment:
Each function should accomplish one thing only, There seems to be a lot going on in this function. I recommend it is renamed to convert_pet and attempts to only convert a DICOM PET to a Nifti. 

We deal with RTStructs etc in other parts of the library.